internal/tools/go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/fatih/color v1.18.0 // indirect
 	github.com/fatih/structtag v1.2.0 // indirect
 	github.com/firefart/nonamedreturns v1.0.5 // indirect
-	github.com/fsnotify/fsnotify v1.6.0 // indirect
+	github.com/fsnotify/fsnotify v1.8.0 // indirect
```

Comment:
Just curious: why this upgrade?

Reply:
I think this must have gotten updated with my original (since removed) changes, I'll revert.